grails-app/services/com/unifina/service/SessionService.groovy

```diff
@@ -14,9 +14,17 @@ class SessionService {
 
 	KeyValueStoreService keyValueStoreService
 
+	void updateUsersLoginDate(SecUser user, Date date) {
```

Comment:
Can be `private` or even removed completely, 2 line function and used in only one place

Reply:
I'll leave it as it is. there's a test for it.